migrations/versions/6b6e42ec28c0_total_number_of_projects_added.py

```diff
@@ -0,0 +1,28 @@
+"""total_number_of_projects_added
+
+Revision ID: 6b6e42ec28c0
+Revises: b976f6cda95c
+Create Date: 2023-05-29 08:50:56.171867
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "6b6e42ec28c0"
+down_revision = "b976f6cda95c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("reporting", sa.Column("total_projects_count", sa.Integer(), nullable=False))
```

Comment:
Since we already have rows in the table we need them either to have a value or to be nullable. In this case nullable=True is more representative.
```suggestion
    op.add_column("reporting", sa.Column("total_project_count", sa.Integer(), nullable=True))
```